app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/completeregistration/CompleteRegistrationViewModel.kt

```diff
@@ -215,10 +218,8 @@ class CompleteRegistrationViewModel @Inject constructor(
             }
 
             is RegisterResult.Success -> {
-                mutableStateFlow.update { it.copy(dialog = null) }
-                sendEvent(
-                    CompleteRegistrationEvent.NavigateToLogin(
-                        email = state.userEmail,
+                trySendAction(
+                    action = Internal.AttemptLogin(
```

Comment:
Why doe we need this event?

Can we just launch the scope here and then send the result event?

Reply:
ok so basically opposite of the change I made from the first commit, same amount of chaining just different order, attemp the login here, send result through action channel so state is changes on that scope and bob's your uncle?